.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@ See: https://jira.bf.local/browse/ECR-XYZ
 
 ### Definition of Done
 
-- [ ] There are no TODOs left in the code
+- [ ] There is no TODOs left in the code
```

Comment:
@Maria-Nosyk , I think we need your help :-) 

As there might be multiple TODOs, don't we have to use 'are'?